src/screens/Shared/EditLocation.js

```diff
@@ -40,7 +40,7 @@ const AddLocationCallout = () => (
 class EditLocation extends Component {
   state = { inputValue: "", results: [] };
 
-  geocode = async address => {
+  displayAddress = async address => {
```

Comment:
This is just a semantic thing, but I like to name my functions with verbs, so I'd name it something like "setDisplayAddress" to make it clear that the 'display' indicates that this address is the one that is shown to the user (as opposed to its meaning as a verb).